swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
+
 class Item:
-    pass
\ No newline at end of file
+    category = ""
+    
+    def __init__(self, category = category, condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        self = "Hello World!"
+        return f"{self}"
+
+    def condition_description(self):
```

Comment:
This logic will handle anything unexpected below the "normal" range (anything 1 or less will be trash), but what would happen for conditions greater than 5? Maybe the behavior here is OK, but be sure to think about those scenarios.

Also, notice how repetitive all the conditions are. If we wanted to add additional conditions, we would need to write more code to handle them. What if we stored the conditions in a collection and could somehow calculate where in the collection the appropriate description for each condition could be found? If we do it carefully, adding additional conditions could be as simple as adding a new string to the collection, and there might be less code overall that would be easier to test (are ALL of these messages currently being tested?).